UI/Panels/Device/MFMultiplexerDriverSubPanel.resx

```diff
@@ -196,7 +196,7 @@
     <value>32</value>
   </data>
   <data name="labelMaster.Text" xml:space="preserve">
-    <value>Changing these values will affect every other device using multiplexers.</value>
+    <value>Changing these values will affect every other device in this module using multiplexers.</value>
```

Comment:
Do we want to say:
Changing these values will affect all other multiplexers of this module.

Reply:
> Do we want to say: Changing these values will affect all other multiplexers of this module.
I don't really see much of a difference meaning-wise, but I like the fact that it's shorter.
> it sounds almost like a warning. but it is not.
But it _is_ a warning. That's why it has been placed there.
> These values will be shared by all multiplexers of this module.
That's another equivalent way to put it. But it just seems like a lazy reminder, which doesn't alert the user that he might be doing something unintended - which defeats its purpose.

Reply:
i disagree. It's not supposed to be a warning, it's supposed to be a reminder, a hint. That's why i like the last option best.

These values will be shared by all multiplexers of this module.